client/my-sites/email/email-providers-comparison/index.jsx

```diff
@@ -251,6 +241,27 @@ class EmailProvidersComparison extends Component {
 		}
 	};
 
+	onForwardingConfirmNewMailboxes = () => {
```

Comment:
This is not called anywhere. Before you go ahead and remove this method, make plans to ensure we are sending the `calypso_email_providers_add_click` event with a valid `source` for forwards, so it can be consistent with what we do for Titan and Google.